lib/tasks_private/spec_helper.rake

```diff
@@ -204,6 +243,24 @@ class PopulateTower
       count = JSON.parse(@conn.get("/api/v1/#{tower_name}/").body)['count']
       puts("%s %s: %s" % [miq_name.ljust(30), "(#{tower_name})".ljust(20), count])
     end
-    self
+  end
+
+  def information
+    return unless @organization
+
+    puts "=== Further instructions ==="
+
+    credentials_add_uri = ENV['TOWER_URL'].chomp('api/v1/') + '#/credentials/add/'
+    puts <<~"INFORMATION"
+      Vault credential cannot be created via API.
```

Comment:
`vault` credential is using `kind='ssh'` with only the `vault_password` field (and `name`) filled.

Reply:
Oh! Thanks for that information. I’ll try it out and replace the instruction with actual creation of the credential.